sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -76,7 +76,30 @@ extern "C" {
  *          IANA, IP TIME TO LIVE PARAMETER
```

Comment:
Why? There it is defined, that the default IP TTL (IPv4 terminology for Hop Limit) is 64.